apps/web/src/App.tsx

```diff
@@ -180,12 +193,142 @@ function App() {
+                                      navigateTo: '/quickstart/notification-center',
+                                    },
+                                    {
+                                      navIcon: NestedSquares,
+                                      description: 'Other',
+                                      navigateTo: '/general-started',
+                                    },
+                                  ]}
+                                />
+                              </QuickStartHeader>
+                            </SpotLight>
+                          </RequiredAuth>
+                        }
+                      />
+                      <Route
+                        path="/quickstart/notification-center/set-up"
+                        element={
+                          <RequiredAuth>
+                            <SpotLight>
+                              <QuickStartHeader title={welcomeDescription} description={<ImplementationDescription />}>
+                                <QuickStartCards
+                                  cells={[
+                                    {
+                                      imagePath: `/static/images/frameworks/dark/react.png`,
+                                      navigateTo: '/quickstart/notification-center/set-up/react',
+                                    },
+                                    {
+                                      imagePath: `/static/images/frameworks/dark/angular.png`,
+                                      navigateTo: '/quickstart/notification-center/set-up/angular',
+                                    },
+                                    {
+                                      imagePath: `/static/images/frameworks/dark/vue.png`,
+                                      navigateTo: '/quickstart/notification-center/set-up/vue',
+                                    },
+                                  ]}
+                                />
+                              </QuickStartHeader>
+                            </SpotLight>
+                          </RequiredAuth>
+                        }
+                      />
+                      <Route
+                        path="/quickstart/notification-center/set-up/:framework"
+                        element={
+                          <RequiredAuth>
+                            <SpotLight>
+                              <QuickStartHeader
+                                title={'Pending Notification Center Initialization'}
+                                description={<TroubleshootingDescription />}
+                              >
+                                <SetUp />
+                              </QuickStartHeader>
+                            </SpotLight>
+                          </RequiredAuth>
+                        }
+                      />
+                      <Route
+                        path="/quickstart/notification-center/trigger"
```

Comment:
perhaps we can rename i to a better one, on this page we trigger event + pointing the user to the templates page